en/console-and-shells.rst

```diff
@@ -648,21 +648,27 @@ The methods that allow chaining are:
 - addSubcommand()
 - addSubcommands()
 - command()
+- setCommand()
 - description()
-- epilog()
+- setDescription()
+- setEpilog()
 
-.. php:method:: description($text = null)
+Set the Description
+~~~~~~~~~~~~~~~~~~~
 
-Gets or sets the description for the option parser. The description displays
-above the argument and option information. By passing in either an array or a
-string, you can set the value of the description. Calling with no arguments will
-return the current value::
+.. php:method:: setDescription($text)
+
+The description displays above the argument and option information. By passing
+in either an array or a string, you can set the value of the description.
+Calling with no arguments will return the current value::
 
     // Set multiple lines at once
+    $parser->setDescription(['line one', 'line two']);
+    // Prior to 3.4
```

Comment:
Should be the other way around. (e.g. ::versionadded 3.4 not // Prior to 3.4)

If a whole chapter changes radically why not create a chapter called "console-and-shell (3.4+)" and keep the old chapter as it is... not optimal either but if I got it right people on < 3.4 will be confused by this docs update.